src/Controllers/LoginController.php

```diff
@@ -12,9 +12,15 @@ class LoginController extends BaseController
 
     /**
      * Displays the form the login to the site.
+     *
+     * @return
```

Comment:
```suggestion
     * @return RedirectResponse|string
```
And `use CodeIgniter\HTTP\RedirectResponse;` is needed.

Reply:
**Thanks**, done.